### PR TITLE
Adding new collections to backup script

### PR DESCRIPTION
**Pull request by teone**
**Describe the PR**
Backup all the collections in the firestore DB

### CODE REVIEW

**Review by hwchiu — APPROVED** [review]

File: functions/lib/backup.js [hunk 1]
```diff
@@ -18,9 +18,12 @@ function Backup (bucketName) {
       name: databaseName,
       outputUriPrefix: bucket,
       collectionIds: [
-        'whitelists',
+        'addendums',
         'agreements',
-        'addendums'
+        'appUsers',
+        'contributions',
```
Comment (ccascone) [inline functions/lib/backup.js:24]: Contributions and events contain ephemeral data and they should not be backup up (unless we want to pay more for storage).

File: functions/lib/backup.js [hunk 2]
```diff
@@ -18,9 +18,12 @@ function Backup (bucketName) {
...
+        'events',
+        'whitelists'
```
Comment (ccascone) [inline functions/lib/backup.js:26]: I know this one was not added by this change, but there should be no need to backup whitelists. This collection is generated/updated automatically based on agreement and addendum data. We do update that after every addendum event and every 15 minutes (or so) to be robust against one-off errors.

It is also true that the size here is negligible wrt to storage pricing, so we might as well leave it...

**Review by ccascone — CHANGES_REQUESTED** [review]

File: functions/lib/backup.js (hunk 1 above)
Comment (ccascone) [reply functions/lib/backup.js:24]: The events collection is huge and mostly useless... I'll open a separate github issue for that.

Comment (teone) [reply functions/lib/backup.js]: Done

**Review by ccascone — APPROVED** [review]
LGTM

**Pull request merged**